cms/envs/common.py

```diff
@@ -2116,3 +2116,9 @@
     'country': 'hidden',
 }
 EDXAPP_PARSE_KEYS = {}
+
+######################### FIRST PARTY AUTH ###############################
+
+# If this switch is enabled then `edx.org` users must be loggedin using
+# their `edx.org` google account
+ENABLE_LOGIN_USING_GOOGLE_AUTH = 'enable_login_using_google_auth'
```

Comment:
I think we should create a `waffle.py` in user_authn` app and define waffle switch there instead of polluting settings. something like [this](https://github.com/edx/edx-platform/blob/1754dc511920d493776bd6dc5268f7f67670c807/openedx/core/djangoapps/user_api/config/waffle.py)